environments/storage/file-uploader/.gitignore

```diff
@@ -0,0 +1,22 @@
+
+
+#amplify-do-not-edit-begin
```

Comment:
Is this environment in its own account?

Reply:
Yes, I created a new account `aws-amplify-ui+e2e-storage@amazon.com `